server/auth/auth.go

```diff
@@ -180,19 +181,25 @@ func (client *clientInfo) respHandler(id uint64) *respHandler {
 	return handler
 }
 
+type feeAsset struct {
+	confs uint32
+	amt   uint64
+}
+
 // AuthManager handles authentication-related tasks, including validating client
 // signatures, maintaining association between accounts and `comms.Link`s, and
 // signing messages with the DEX's private key. AuthManager manages requests to
 // the 'connect' route.
 type AuthManager struct {
 	storage        Storage
 	signer         Signer
-	regFee         uint64
 	checkFee       FeeChecker
-	feeConfs       int64
 	miaUserTimeout time.Duration
 	unbookFun      func(account.AccountID)
 
+	feeAddress func(assetID uint32) string
+	feeAssets  map[uint32]*feeAsset
```

Comment:
Good question.  I went back and forth on that too.  My reasoning isn't terribly solid, so I think I'll just switch to the msgjson type.

Reply:
Ah. I see the reasoning now. It's unfortunate to have the string map indices in e.g. `msgjson.ConfigResult.RegFees`, but it makes sense because JSON, and any alternative is uglier. It ultimately won't be an issue, since the symbol is unique enough here.